.github/settings.yml

```diff
@@ -15,15 +15,15 @@ repository:
   # By changing this field, you rename the repository.
 
   # Uncomment this name property and set the name to the current repo name.
-  # name: ""
+  name: "terraform-ibm-icd-elasticsearch"
 
   # The description is displayed under the repository name on the
   # organization page and in the 'About' section of the repository.
 
   # Uncomment this description property
   # and update the description to the current repo description.
-  # description: ""
+  description: "Implements an instance of the IBM Cloud Databases for Elasticsearch service."
 
   # Uncomment this topics property
   # and add a comma-separated list of topics to set on the repo.
-  # topics: terraform, ibm-cloud, terraform-module
+  topics: core-team, terraform, ibm-cloud, terraform-module, ibm-database, cbr-rule
```

Comment:
remove `cbr-rule` and please add `icd-elasticsearch` `elasticsearch` `supported` `graduated`